README.md

```diff
@@ -564,6 +564,7 @@ Simple deployment of [E-mail](https://en.wikipedia.org/wiki/Email) servers, e.g.
 - [Thredded](https://github.com/thredded/thredded) - Forums, feature-rich and simple. `MIT` `Ruby`
 - [Tokumei](https://tokumei.co/) - Anonymous microblogging platform. ([Source Code](https://gitlab.com/tokumei/tokumei)) `ISC` `rc`
 - [twister](http://twister.net.co/) - Fully decentralized P2P microblogging platform leveraging the free software implementations of Bitcoin and BitTorrent protocols. ([Source Code](https://github.com/miguelfreitas/twister-core)) `MIT` `C++`
+- [qpixel](https://codidact.com/) - Q&A-based community knowledge-sharing software . ([Source Code]([https://github.com/codidact/qpixel])) `AGPL-3.0` `Ruby`
```

Comment:
- `([Source Code]([https://github.com/codidact/qpixel]))` should be `([Source Code](https://github.com/codidact/qpixel))`
- Please remove the space at the end of `Q&A-based community knowledge-sharing software .`
-

Reply:
done